### DISCUSSION TITLE
Difference between DeepEdit and Segmentation model

### DISCUSSION THREAD

**Discussion by ArpanGyawali** [Project-MONAI/MONAILabel#1447] [Q&A] [answered]
I have to do the segmentation of the brain from the CTA data. I choose MONAI Label for that, and I am using the Monai extension in the 3D slicer. I have gone through all the lecture videos on YouTube as well as the documentation, but I couldn't find in which case which model would be better to use. To my knowledge, the Segmentation model was introduced later in Monai, so I couldn't find much about it. DeepEdit model uses either UNETR or DynUNet network while the segmentation model uses SegResNet network. But I don't know how they work. Before I start looking at them in deep, can someone tell me the differences in using those models in the Monai label for segmentation, especially for brain CTA?

**Comment by OmarAshkar** [comment]
+1 Additionally, I am wondering how deepedit interaction do work. I see we can add foreground and background points instead of using "grow from seed" but it doesn't really work out for me.

**Reply by diazandr3s** [reply, marked as answer]
Hi @ArpanGyawali,

These are all very good questions. 

DeepEdit is an interactive method that includes the power of two models: the automatic segmentation model and the interactive model. It was designed to facilitate the segmentation while using sophisticated active learning algorithms. However, it needs more GPU memory than the Segmentation model as it works on the whole volume instead of patches as the Segmentation method. 

Here you can find more information about the DeepEdit method: https://arxiv.org/abs/2305.10655

With regards to the network architecture, both the Segmentation and the DeepEdit can use any Transformer based or convolutional-based network.

I'd recommend you start with the Segmentation method first and see how that goes. It needs less GPU and you can test quickly the different hyperparameters. Then you can move to the DeepEdit if needed.

Hope this helps,
